### ISSUE TITLE
An illegal memory access was encounteredan illegal memory access was encountered

### ISSUE THREAD

**Issue by MnogoZnaet** [sadeepj/crfasrnn_keras#49]
Tensorflow 1.11.0
Cuda 9.0
Python 2.7.15

`an illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredCuda kernel failed. Error: an illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredCuda kernel failed. Error: an illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredCuda kernel failed. Error: an illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredCuda kernel failed. Error: an illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredCuda kernel failed. Error: an illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredCuda kernel failed. Error: an illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredCuda kernel failed. Error: an illegal memory access was encounteredCuda kernel failed. Error: an illegal memory access was encounteredCuda kernel failed. Error: an illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredCuda kernel failed. Error: an illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredCuda kernel failed. Error: an illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredCuda kernel failed. Error: an illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredCuda kernel failed. Error: an illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredCuda kernel failed. Error: an illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredCuda kernel failed. Error: an illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredan illegal memory access was encounteredCuda kernel failed. Error: an illegal memory access was encounteredan illegal memory access was encountered2018-11-06 02:40:13.266582: E tensorflow/stream_executor/cuda/cuda_dnn.cc:353] Could not create cudnn handle: CUDNN_STATUS_INTERNAL_ERROR`

**Comment by MnogoZnaet** [comment]
I added this to run_demo.py before main function, and it starts work

```
import tensorflow as tf
from keras.backend.tensorflow_backend import set_session
config = tf.ConfigProto()
config.gpu_options.per_process_gpu_memory_fraction = 0.7
set_session(tf.Session(config=config))
```

**Comment by gyheart** [comment]
> I added this to run_demo.py before main function, and it starts work
> 
> ```
> import tensorflow as tf
> from keras.backend.tensorflow_backend import set_session
> config = tf.ConfigProto()
> config.gpu_options.per_process_gpu_memory_fraction = 0.7
> set_session(tf.Session(config=config))
> ```

Did train the CRF layer successfully？ Would you share your training code？
My email is gyimage@yeah.net.
Thanks.

**Comment by cv-shiyanshi** [comment]
@gyheart  Have you got the training code? I have trying to train it with caffe，but something wrong with it,and I am going to train it with tensorflow,could you give me some advice？

**Comment by youguai** [comment]
> @gyheart Have you got the training code? I have trying to train it with caffe，but something wrong with it,and I am going to train it with tensorflow,could you give me some advice？

No， I did'ot train it in keras successfully.

**Comment by gyheart** [comment]
> @gyheart Have you got the training code? I have trying to train it with caffe，but something wrong with it,and I am going to train it with tensorflow,could you give me some advice？

No. I did't train it successfully in keras. If you get the code, please tell me.  Thanks.
